runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomCounter.java

```diff
@@ -47,7 +50,10 @@ public String getText()
 	@Override
 	public String getTooltip()
 	{
+		int topUpAmount = config.maxCofferAmount() - plugin.getCoffer();
+
 		return "Favor: " + plugin.getFavor() + "/127" + "</br>"
-			+ "Coffer: " + QuantityFormatter.quantityToStackSize(plugin.getCoffer());
+			+ "Coffer: " + QuantityFormatter.quantityToStackSize(plugin.getCoffer()) + "</br>"
+			+ "Top up: " + topUpAmount;
```

Comment:
I was thinking of this, but then would it not be better for it to show the exact amount, even if it is a bit uglier? If the topUpAmount was say 73103 it would show as 73.1k.

Reply:
Since the coffer amount is already formatted i think it should be consistent. It's easier to just type 74k than 73103 anyway if you want to be on the edge like that.

Reply:
Fair enough, I see your point. I suppose I'm used to being nit picky and putting in the exact amount, but I can't think of a way to show that in a consistent, 'pretty' way.

Reply:
If nothing else, this should be formatted to include separators for numbers in the thousands and millions. (eg. `7,000` and `7,000,000` instead of `7000` and `7000000`)